apps/salesforce-commerce-cloud/src/components/dialog/ProductPreview.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { Flex, Text } from '@contentful/f36-components';
+
+const ProductPreview = (props: { product: any }) => {
+  const descriptionLength = 178;
+  const { product } = props;
+
+  let description = 'Description';
+  if (product.shortDescription.default) {
+    description =
+      product.shortDescription.default.markup.length > descriptionLength
+        ? `${product.shortDescription.default.markup.substring(0, descriptionLength)}...`
+        : product.shortDescription.default.markup;
+  }
```

Comment:
```suggestion
  const formatShortDescription(productMarkup?: string) {
    if (!productMarkup) {
      return '';
    }

    if (productMarkup.length > descriptionLength) {
      return `${productMarkup.substring(0, descriptionLength)}...`;
    }

    return productMarkup;
  }

  const description = formatShortDescription(product.shortDescription.default?.markup) || 'Description';
```
Changing this over to a const and using a function with a return value could help make this section a little faster to read